src/main.ts

```diff
@@ -23,28 +23,10 @@ export async function run() {
     );
 
     if (versionSpec) {
-      let installDir: string | undefined = tc.find('go', versionSpec);
-
-      if (!installDir) {
-        console.log(
-          `A version satisfying ${versionSpec} not found locally, attempting to download ...`
-        );
-        installDir = await installer.downloadGo(versionSpec, stable);
-        console.log('Installed');
-      }
-
-      if (installDir) {
-        core.exportVariable('GOROOT', installDir);
-        core.addPath(path.join(installDir, 'bin'));
-        console.log('Added go to the path');
+      let token = core.getInput('token');
```

Comment:
As far as I know, to use `core.getInput('token')` you need to specify the 'token' input to the `action.yml` file. Something like this: 
```
inputs: 
......
   token:
    description: Used to pull Go distributions from actions/go-versions. Since there's a default, this is typically not supplied by the user.
    default: ${{ github.token }}
```

Reply:
Done.